src/main/java/dev/snowdrop/release/model/cpaas/release/CPaaSReleaseFile.java

```diff
@@ -0,0 +1,28 @@
+package dev.snowdrop.release.model.cpaas.release;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class CPaaSReleaseFile {
```

Comment:
I tried adding PIG as a library, at the beginning of April, without success. Nevertheless in this case it's the CPaaS configuration and I'll ask them if they'r project is accessible and we can consume these types.

Regarding PNC, this is what I posted in the PNC Users group for which I didn't get any answer. 

> As an update, I haven't been able to import the project as library of our Quarkus application. It seems the problem is in the CDI injections, some classes aren't properly gathered. To reach this conclusion I had some help internally, since I'm far from being a CDI or a Quarkus expert, so I'm not sure how much information I can provide.

IIRC @geoand gave me a hand looking at these CDI injection problems.

Reply:
Maybe we need to "escalate" this a little bit… what was the issue with using PiG, if you recall?

Reply:
It was the CDI problem. IIRC there were classes with some annotation that prevented CDI from getting them.